style.css

```diff
@@ -0,0 +1,109 @@
+* {
+    padding: 0;
+    margin: 0;
+  }
+  
+  body {
+    background-color: lightgray;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+  
+  .container {
+    width: 100%;
+    max-width: 500px;
+    background-color: white;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+  }
+  
+  .container h1 {
+    text-align: center;
+    margin-bottom: 20px;
+  }
+  
+  .container form {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  .container form input {
+    padding: 10px;
+    margin-bottom: 10px;
+    border: none;
+    width: 100%;
+  }
+  
+  #input:focus {
+    outline: none;
+  }
+  
+  .line {
+    height: 1px;
+    background-color: lightgray;
+    margin: 10px 0;
+  }
+  
+  .list {
+    list-style: none;
+  }
+  
+  .clear {
+    text-align: center;
+    margin-top: 10px;
+    cursor: pointer;
+    color: rgb(125, 121, 121);
```

Comment:
- [ ] Hello @asharanjith this color is assigned multiple times throughout your code. I'd suggest that you assign it to a class so whenever you want to change it you'll just change one line.

Reply:
Hi, Thanks for the suggestion. I created a variable name for the color code and made a relevant change to the code